ExtraCalcFunctions.php

```diff
@@ -34,6 +34,15 @@ public function redcap_every_page_before_render( $project_id )
 		$lastDuration = $project_id === null
 		                ? 0 : ( $this->getProjectSetting( 'calc-values-auto-update-dur' ) ?? 0 );
 		$autoCalcWait = $lastDuration < 90 ? 900 : ( $lastDuration * 10 );
```

Comment:
Just wandered if x10 for interval (from 15 minute interval to 2.5 hours) is a bit much if last run duration is over 90 seconds, and it could go up again to 25 hours if next run takes over 90 seconds (unless I've got hold of the wrong end of the stick).  In fact with the new split function, is this even needed at all now?

Reply:
Basically the longer it takes to do the calculations, the longer the interval before it runs again (minimum interval 15 minutes). The split runs will aim to cap run execution at 10 minutes, so the interval should not exceed 100 minutes in practice.